### PR TITLE
Credits

### PR DESCRIPTION
**Pull request by PikalaxALT**
Decompile asm/credits.s and dump relevant data from data/data_83FECCC.s

### CODE REVIEW

File: include/credits.h [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#ifndef GUARD_CREDITS_H
+#define GUARD_CREDITS_H
+
+
```
Comment (jiangzhengwenjz) [inline include/credits.h]: ?

File: include/overworld.h [hunk 2]
```diff
@@ -119,4 +126,7 @@ bool32 sub_8055C9C(void);
 void Overworld_ResetStateAfterDigEscRope(void);
 bool32 sub_8058244(void);
 
+void sub_80574A4(void);
+bool32 sub_805750C(u8 *, const struct CreditsOverworldCmd *, u8);
+
 #endif //GUARD_ROM4_H
```
Comment (jiangzhengwenjz) [inline include/overworld.h]: It seems that someone didn't change the header macro after changing the header file name

File: src/credits.c [hunk 3]
```diff
@@ -0,0 +1,2047 @@
...
+    u16 field_02;
+    u16 field_04;
+    u8 field_06;
+    u16 field_08;
+    u16 field_0A;
+};
+
+EWRAM_DATA struct CreditsResources * gUnknown_203AB40 = NULL;
+
+void sub_80F39E8(void);
```
Comment (jiangzhengwenjz) [inline src/credits.c]: Many should be static.

File: data/strings.s [hunk 4]
```diff
@@ -3323,260 +3323,345 @@ gUnknown_841D18D:: @ 841D18D
 gUnknown_841D198:: @ 841D198
 	.string "Pokémon FireRed Version\nStaff$"
 
-gUnknown_841D1B6:: @ 0x841D1B6
-	.string "  Pokémon LeafGreen Version\nStaff$"
+	.align 2
+gUnknown_841D1B8:: @ 0x841D1B6
```
Comment (jiangzhengwenjz) [inline data/strings.s]: The addresses in commentS are not changed

File: include/credits.h (hunk 1 above)
Comment (PikalaxALT) [reply include/credits.h]: Created just in case but was never needed. Deleted.

File: include/overworld.h (hunk 2 above)
Comment (PikalaxALT) [reply include/overworld.h]: Fixed.

File: src/credits.c (hunk 3 above)
Comment (PikalaxALT) [reply src/credits.c]: Fixed

File: data/strings.s (hunk 4 above)
Comment (PikalaxALT) [reply data/strings.s]: Address comments will be marked for removal in a future task.

**Pull request merged**